Task 5/script.js

```diff
@@ -9,13 +9,19 @@ atveju grąžins masyvą su "users", kurie yra pilnamečiai.
 -------------------------------------------------------------------------- */
 
 const users = [
-  { id: '1', name: 'John Smith', age: 20, hasDog: true },
-  { id: '2', name: 'Ann Smith', age: 24, hasDog: false },
-  { id: '3', name: 'Tom Jones', age: 31, hasDog: true },
-  { id: '4', name: 'Rose Peterson', age: 17, hasDog: false },
-  { id: '5', name: 'Alex John', age: 25, hasDog: true },
-  { id: '6', name: 'Ronald Jones', age: 63, hasDog: true },
-  { id: '7', name: 'Elton Smith', age: 16, hasDog: true },
-  { id: '8', name: 'Simon Peterson', age: 30, hasDog: false },
-  { id: '9', name: 'Daniel Cane', age: 51, hasDog: true },
+  { id: "1", name: "John Smith", age: 20, hasDog: true },
+  { id: "2", name: "Ann Smith", age: 24, hasDog: false },
+  { id: "3", name: "Tom Jones", age: 31, hasDog: true },
+  { id: "4", name: "Rose Peterson", age: 17, hasDog: false },
+  { id: "5", name: "Alex John", age: 25, hasDog: true },
+  { id: "6", name: "Ronald Jones", age: 63, hasDog: true },
+  { id: "7", name: "Elton Smith", age: 16, hasDog: true },
+  { id: "8", name: "Simon Peterson", age: 30, hasDog: false },
+  { id: "9", name: "Daniel Cane", age: 51, hasDog: true },
 ];
+
+const filterDogOwers = () => users.filter((user) => user.hasDog === true);
```

Comment:
Function should take argument and then you should filter through that argument. Right now it always takes `users` variable, but if you would like to run this function with other variable you won't be able to do that without changing inside of the function. Same problem with function below.